<launch>
  <arg name="points_topic" default="points_raw" />
  <arg name="outputed_topic" default="obstacle_points_filtered" />
  <arg name="x_min" default="-50.0" />
  <arg name="x_max" default="50.0" />
  <arg name="y_min" default="-4.5" />
  <arg name="y_max" default="4.5" />
  <arg name="z_min" default="-2.0" />
  <arg name="z_max" default="0.5" />

  <node pkg="jude_percept" name="realtime_lc_filter" type="realtime_lc_filter">
    <param name="points_topic" value="$(arg points_topic)" />
    <param name="output_topic" value="$(arg outputed_topic)" />
    <param name="x_min" value="$(arg x_min)" />
    <param name="x_max" value="$(arg x_max)" />
    <param name="y_min" value="$(arg y_min)" />
    <param name="y_max" value="$(arg y_max)" />
    <param name="z_min" value="$(arg z_min)" />
    <param name="z_max" value="$(arg z_max)" />
  </node>
</launch>
